src/test/compile-fail/issue-13033.rs

```diff
@@ -16,7 +16,9 @@ struct Baz;
 
 impl Foo for Baz {
     fn bar(&mut self, other: &Foo) {}
-    //~^ ERROR method `bar` has an incompatible type for trait: values differ in mutability [E0053]
```

Comment:
it's hard for me to tell from this diff: has the "values differ in mutability" part of the message been entirely removed?